.github/ISSUE_TEMPLATE/05-epic.yml

```diff
@@ -12,3 +12,16 @@ body:
   attributes:
     label: Description
     description: Describe the epic
+- type: dropdown
+  id: estimation
+  attributes:
+    label: Have you provided an initial effort estimate for this issue?
+    description: See our [handbook](https://flowforge.com/handbook/development/releases/planning/#effort-estimation) for more details.
+    multiple: false
+    options:
+      - I have provided an initial effort estimate
+      - I am no FlowForge team member
+      - I can not provide an initial effort estimate
+      - All subtasks of this epic have an effort estimate
```

Comment:
At the point in time the epic is raised we won't expect to have all the subtasks already raised and sized. So this option will never be used when raising the epic. Once raised, I'm not sure if the template carries over to be edited later...

Reply:
Agree, I will change it to a hint. 

Reply:
[changed dropdown to hint for epics](https://github.com/flowforge/flowforge/pull/2046/commits/3b45e184418ff1dd62761b9079ea224dc75d87e4)